packages/components/storybook/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+	"presets": [ "@wordpress/babel-preset-default" ],
+	"plugins": [ "babel-plugin-emotion" ]
```

Comment:
This file was moved to `storybook/.babelrc` - this one should be removed and the other one updated. It's a merge issue.

Reply:
Gotcha! Thank you for spotting